nix/README.rst

```diff
@@ -30,9 +30,12 @@ Once you have a working Nix installation use::
     $ nix-shell --pure
 
 in the base :term:`FiPy` directory to install :term:`FiPy` with
-Python 2. To install with Python 3, use::
+Python 2 by default. To use with another version of Python use
 
-    $ nix-shell --pure nix/py3.nix
+    $ nix-shell --pure --argstr python_version xy
+
+for version x.y of Python. Currently, only tested with versions 2.7,
+3.6 and 3.7.
```

Comment:
Still [fails for me on macOS with python_version 36 or 37]( https://gist.github.com/guyer/f70a2a41f0e3b2b30db86dd831e747a3)

Reply:
default (27)  fails now, too

Reply:
Oh Lord. I don't have a clue about the error messages. I'll need a Mac to try and debug.